packages/migrate/src/__tests__/fixtures/introspection/postgresql/views-warning-26/setup.sql

```diff
@@ -0,0 +1,5 @@
+-- TODO: remove tmp table once we no longer receive error P4001
```

Comment:
I think this should already be resolved, right @pimeys?

Reply:
Yep, seems to be!